discord_thread_summarizer_using_openai/ai_utils.bal

```diff
@@ -0,0 +1,23 @@
+import ballerinax/openai.chat;
+
+configurable string openAIToken = ?;
+
+chat:Client chatClient = check new ({
+    auth: {
+        token: openAIToken
+    }
+});
+
+function generateChatCompletion(string prompt, string model = "gpt-3.5-turbo") returns string|error {
+    chat:CreateChatCompletionRequest req = {
+        model: model,
```

Comment:
```suggestion
        model,
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/1dbf76e067fd08cc0430182aa02b7c98d154f3ac